src/main/java/bit/Storage.java

```diff
@@ -0,0 +1,170 @@
+package bit;
+
+import java.io.*;
+
+public class Storage {
+
+    private final String FILE_PATH = "./data/bit.txt";
+    private final File FILE = new File(FILE_PATH);
+
+    public Storage() {
+        boolean isCreated = createFile(FILE);
+
+    }
+
+    /**
+     * Create a file myFile if it does not exist. If file exists after this method, return true.
+     * Otherwise, return false.
+     * @param myFile file to be created
+     * @return true or false based on whether file exists after method is run.
+     */
+    public boolean createFile(File myFile) {
+        try {
+            if (!myFile.exists()) {
+                myFile.getParentFile().mkdirs();
+                myFile.createNewFile();
+            }
+            return true;
+        } catch (IOException e) {
+            System.out.println(e);
+            return false;
+        }
+
+    }
+
+    /**
+     * Load all tasks stored in text file to Tasklist.
+     * @param list list of tasks to be loaded
+     */
+    public void loadFile(Tasklist list) {
+
+        try (BufferedReader br = new BufferedReader(new FileReader(FILE_PATH))) {
+            String next;
+            while ((next = br.readLine()) != null) {
+                String[] parts = next.split("/");
+                if (parts.length == 1) {
+                    return;
+                }
+                System.out.println(parts[0]);
+                System.out.println(parts[1]);
+                if (parts[0].equals("T")) {
+                    Task t = new Todo(parts[1]);
+                    if (parts[2].equals("M")) {
+                        t.complete();
+                    }
+                    list.addFromStorage(t);
+                } else if (parts[0].equals("D")) {
+                    Task d = new Deadline(parts[1], parts[2]);
+                    if (parts[3].equals("M")) {
+                        d.complete();
+                    }
+                    list.addFromStorage(d);
+                } else if (parts[0].equals("E")) {
+                    Task e = new Event(parts[1], parts[2], parts[3]);
+                    if (parts[4].equals("M")) {
+                        e.complete();
+                    }
+                    list.addFromStorage(e);
+                }
+            }
+        } catch (IOException e) {
+            System.out.println("Errorrrr...");
+        }
+    }
+
+    /**
+     * Save all tasks in the list to a text file.
+     * @param tasklist the tasks to be stored.
+     */
+    public   void saveAll(Tasklist tasklist) {
```

Comment:
I think you accidentally added extra spaces after 'public'. Also saw it in other methods too :)

Reply:
I don't know how that happened lol. 😅